src/crypto/PSAOperationalKeystore.h

```diff
@@ -19,13 +19,29 @@
 
 #include <crypto/CHIPCryptoPALPSA.h>
 #include <crypto/OperationalKeystore.h>
+#include <lib/core/CHIPPersistentStorageDelegate.h>
 
 namespace chip {
 namespace Crypto {
 
 class PSAOperationalKeystore final : public OperationalKeystore
 {
 public:
+    /**
+     * @brief Initialize the PSA Operational Keystore and move all sensitive keys from storage to ITS secure storage.
+     *
+     * After moving all sensitive keys to ITS, the old one stored in the storage will be removed and not available anymore.
```

Comment:
This needs to more clearly document that this can fail partway through, after having moved some of the keys over, right?  And what the exact behavior is in that case.

Reply:
I've changed the way to migrate those keys. Now the keys are migrated on the HasOpKeypairForFabric method call, so in fact on CASE session establishment. Thanks to that when one of the migrations finished with an error, the other ones for other fabrics will be working and it won't break the device, like previously.